content/integrate/redis-data-integration/ingest/installation/_index.md

```diff
@@ -88,6 +88,147 @@ Once the installation is finished, RDI is ready for use.
 
 {{<note>}}RDI gives you instructions to help you create secrets and create your pipeline.{{</note>}}
 
+## "Silent" installation
+
+You can use the
+[`redis-di install`]({{< relref "/integrate/redis-data-integration/ingest/reference/cli/redis-di-install" >}})
+command with the `--file` option (or the shorter version `-f`) to supply answers
+to the installer's questions automatically from a [TOML](https://toml.io/en/) file:
+
+```bash
+redis-di install --file silent.toml
+```
+
+The following example shows the properties you can use:
+
+```toml
+title = "RDI Silent Installer Config"
+
+high_availability = false
+scaffold = false
+deploy = true
+db_index = 4
+deploy_directory = "/opt/rdi/config"
+
+[rdi.cluster]
+host = "localhost"
+port = 9443
+username = "username"
+password = "password"
+
+[rdi.database]
+host = "localhost"
+port = 12001
+username = "username"
+password = "password"
+use_existing_rdi = true
```

Comment:
this is when you don't need the cluster credentials nor the high availability because you specify existing RDI database

Reply:
Fixed.